tapgarden/caretaker.go

```diff
@@ -776,11 +798,15 @@ func (b *BatchCaretaker) stateStep(currentState BatchState) (BatchState, error)
 
 		// At this point we have a fully signed PSBT packet which'll
 		// create our set of assets once mined. We'll write this to
-		// disk, then import the public key into the wallet.
+		// disk, then import the public key into the wallet. The sibling
+		// here can always be nil as we'll fetch the output key computed
+		// in BatchStateFrozen.
 		//
 		// TODO(roasbeef): re-run during the broadcast phase to ensure
 		// it's fully imported?
-		mintingOutputKey, tapRoot, err := b.cfg.Batch.MintingOutputKey()
+		mintingOutputKey, tapRoot, err := b.cfg.Batch.MintingOutputKey(
+			nil,
```

Comment:
If we make the parameter not a pointer, we can just pass in an empty params object with just `None()` for the two values.